lib/tessel/provision.js

```diff
@@ -256,7 +287,7 @@ function AlreadyAuthenticatedError() {
 util.inherits(AlreadyAuthenticatedError, Error);
 
 actions.AlreadyAuthenticatedError = AlreadyAuthenticatedError;
-
+module.exports.TESSEL_AUTH_KEY = filepath;
```

Comment:
Also, you will need to place this line below the next because `module.exports = actions` overwrites this variable.